api/api.py

```diff
@@ -294,3 +294,64 @@ def custom_build(ref: str, command: str, deps: List[str], tag: str = "", disable
     disable_push: whether Tilt should push the image in to the registry that the Kubernetes cluster has access to. Set this to true if your command handles pushing as well.
   """
   pass
+
+class LiveUpdateStep:
+    """A step in the process of performing a LiveUpdate on an image's container.
+
+    See :meth:`live_update`.
+    """
+    pass
+
+def workdir(dir: str) -> LiveUpdateStep:
```

Comment:
also sync "dest"'s, if relative, are evaluated relative to this. maybe worth a "see also" to docker's WORKDIR command?

Reply:
specify that we only allow one and it must come first

Reply:
oh hmm you list this below... idk, maybe worth listing here as well? (and similarly under container_restart?) 🤷‍♀️